discord-scripts/thread-management/auto-join.ts

```diff
@@ -46,13 +46,22 @@ async function autoJoinThread(
 
   // Use this to assign a specific role based on the mapping in CUSTOM_CHANNEL_ROLE, in order to map specific roles/channels
   if (hasCustomChannels && containingChannel) {
-    const roleName = CUSTOM_CHANNEL_ROLE[containingChannel.name]
-    if (roleName) {
-      const channelMatchingRole = server.roles.cache.find(
-        (role) => role.name.toLowerCase() === roleName.toLowerCase(),
-      )
-      if (channelMatchingRole) {
-        await placeholder.edit(channelMatchingRole.toString())
+    const roleNames = CUSTOM_CHANNEL_ROLE[containingChannel.name]
+      ?.split(",")
+      .map((role) => role.trim())
```

Comment:
This isn't a user input so no need to `trim`. If we allow users to set this manually later, we should `trim` when they give us a role name, rather than having to do it every time we check for matching channels.